net/developer-guide/slides-for-net-6-cross-platform/_index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Aspose.Slides for .NET 6 Cross Platform
+type: docs
+weight: 237
+url: /net/slides-for-net-6-cross-platform
+keywords: "Aspose.Slides", ".NET", "Cross platform"
+description: "Aspose.Slides for .NET 6 Cross Platform"
+---
+
+1. Cross-platform Aspose.Slides for .NET6 can be used for .NET 7 and future .NET releases.
+
+2. **Prerequisite**: To use the cross-platform version Aspose.Slides for .NET 6, you need to download the Aspose.Slides package from the [Release Page](https://releases.aspose.com/slides/net/). The Aspose.Slides on NuGet is not suitable because it provides cross-platform support for only the .NET Standard.
+
+## **Get and Use Cross-Platform Aspose.Slides**
+
+1. Download the ZIP package of the latest Aspose.Slides release from the [Release Page](https://releases.aspose.com/slides/net/). 
+
+2. Unpack the files from *\Aspose.Slides.zip\Aspose.Slides\net6.0\crossplatform* and place them in the folder that will be used for dependencies in your project.
+
+3. Add a reference to Aspose.Slides.dll
+
+   In our example (below), the libraries are located in the project folder along this path: ConsoleApp\libs\Aspose.Slides\net6.0\crossplatform\...
+
+4. Place the remaining files that Aspose.Slides depend in the output directory by adding appropriate instructions to the csproj project file this way:
+
+   ​        <ItemGroup>
```

Comment:
@EmSeriki ,

It seems that when viewed formatted, this text is lost? Perhaps an escape is needed.